src/containerapp/azext_containerapp/azext_metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67"
```

Comment:
Why the `azext.minCliCoreVersion` is so low?

Reply:
Updated to 2.15.0